src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -24,6 +27,10 @@
 public class MainWindow extends UiPart<Stage> {
 
     private static final String FXML = "MainWindow.fxml";
+    private static final Double PERSON_LIST_RATIO = 0.25;
+    private static final Integer MINIMUM_HEIGHT = 700;
+    private static final Integer MINIMUM_WIDTH = 700;
+    private Image logo = new Image(this.getClass().getResourceAsStream("/images/logo.png"));
```

Comment:
Minor nitpick but do you think renaming `logo.png` to `friendfolio_logo.png` will be better because the naming less vague?